spec/features/review/user_adds_review_spec.rb

```diff
@@ -45,4 +45,16 @@
 
     expect(page).to_not have_link("Create Review")
   end
+
+  scenario "I can see picture I add when I write a review" do
+    restaurant = FactoryGirl.create(:restaurant)
+    reviewer = FactoryGirl.create(:user)
+    review = FactoryGirl.create(:review)
+    sign_in_as(reviewer)
+    visit restaurant_path(review.restaurant)
+
+    within ".review-image" do
+      expect(page).to have_css("img")
+    end
+  end
```

Comment:
When are you adding a picture here? Also, you aren't using `restaurant` so get rid of it. Since you have associations in place in your Factories, you can call `review.restaurant` which would return a restaurant object associated with `review`.